### PR TITLE
Story update #1

### PR DESCRIPTION
**Pull request by chatsagnik**

### CODE REVIEW

**Review by jayantbh — CHANGES_REQUESTED** [review]
Indescriptive PR name.

File: editor.c [hunk 1]
```diff
@@ -52,3 +58,5 @@ int main(int argc, char *argv[])
 	fclose(file);
 	return 0;
 }
+
+// Green hoti Cabbage, Modi is savage xD
```
Comment (jayantbh) [inline editor.c]: Missing newline. :P xD

Comment (chatsagnik) [reply editor.c]: I am not allowed to abuse here right? :p

Comment (jayantbh) [reply editor.c]: No. xD 
The missing newline symbol should be an indication that there's supposed to be a newline here.

**Review by jayantbh — APPROVED** [review]

**Pull request merged**